src/components/Reactions/AddReactionBubble.js

```diff
@@ -70,7 +81,7 @@ const AddReactionBubble = (props) => {
             <Pressable
                 ref={ref}
                 style={({hovered, pressed}) => [styles.emojiReactionBubble, styles.userSelectNone, StyleUtils.getEmojiReactionBubbleStyle(hovered || pressed, false, props.isContextMenu)]}
-                onPress={onPress}
+                onPress={Session.checkIfActionIsAllowed(props.session.authTokenType, onPress)}
```

Comment:
I'm a bit concerned on how well this will scale. Are we gonna have to keep adding this wrapper in a bunch of new developed features?

Reply:
Yeah, that's true. This is something I didn't realize until the implementation phase. I agree that we'd need to come up with a more generic solution. Given the deadline for this I'm going to work on that as [follow-up](https://github.com/Expensify/Expensify/issues/286831).